packages/@okta/vuepress-site/docs/guides/protect-your-api/main/go/configureapi.md

```diff
@@ -5,6 +5,8 @@
    OKTA_API_AUDIENCE={yourAudience}
    ```
 
+    >**Note:** If you're using a custom authorization server other than `default`, use the authorization server `id` in place of the `{yourAuthServerName}` placeholder.
```

Comment:
I suggest adding a quick example of what a custom auth server name would look like and how to obtain the "ID" with the given server name,

Reply:
Added an example `id` and xref to the List all auth servers API